src/types.tsx

```diff
@@ -14,7 +14,8 @@ export class DdSdkConfiguration {
     readonly applicationId: string,
     readonly nativeCrashReportEnabled: boolean,
     readonly sampleRate: number,
-    readonly additionalConfig: object = {}
```

Comment:
we need to either put it back or modify this line https://github.com/DataDog/dd-sdk-reactnative/blob/abfd9a7ecb2b231f5e8d1c657219caa302702cad/src/DdSdkReactNative.tsx#L35